nodebuilder/node/node.go

```diff
@@ -23,18 +24,18 @@ type Module interface {
 	// AuthVerify returns the permissions assigned to the given token.
 	AuthVerify(ctx context.Context, token string) ([]auth.Permission, error)
 	// AuthNew signs and returns a new token with the given permissions.
-	AuthNew(ctx context.Context, perms []auth.Permission) (string, error)
+	AuthNew(ctx context.Context, perms []auth.Permission, ttl time.Duration) (string, error)
 }
 
 var _ Module = (*API)(nil)
 
 type API struct {
 	Internal struct {
-		Info        func(context.Context) (Info, error)                                `perm:"admin"`
-		Ready       func(context.Context) (bool, error)                                `perm:"read"`
-		LogLevelSet func(ctx context.Context, name, level string) error                `perm:"admin"`
-		AuthVerify  func(ctx context.Context, token string) ([]auth.Permission, error) `perm:"admin"`
-		AuthNew     func(ctx context.Context, perms []auth.Permission) (string, error) `perm:"admin"`
+		Info        func(context.Context) (Info, error)                                                   `perm:"admin"`
+		Ready       func(context.Context) (bool, error)                                                   `perm:"read"`
+		LogLevelSet func(ctx context.Context, name, level string) error                                   `perm:"admin"`
+		AuthVerify  func(ctx context.Context, token string) ([]auth.Permission, error)                    `perm:"admin"`
+		AuthNew     func(ctx context.Context, perms []auth.Permission, ttl time.Duration) (string, error) `perm:"admin"`
```

Comment:
API break, need to update PR description

Reply:
Reverted